<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
<<<<<<< HEAD
    <item row="4" column="0" colspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Встать в очередь</string>
      </property>
     </widget>
    </item>
=======
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLineEdit" name="lineEdit"/>
        <widget class="QLineEdit" name="lineEdit_2"/>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Встать в очередь</string>
        </property>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser"/>
     </widget>
    </item>
>>>>>>> b0a19df139c766a16cc9d28ee812b1d181e94553
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
<<<<<<< HEAD
     <height>26</height>
=======
     <height>25</height>
>>>>>>> b0a19df139c766a16cc9d28ee812b1d181e94553
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
